app/assets/stylesheets/_responsive.css.scss

```diff
@@ -6,8 +6,34 @@
   nav .inner-left {
     @include span-columns(3 of 5);
 
+    .mobile-control {
+      cursor: pointer;
+      display: inline-block;
+      padding-right: 10px;
+    }
+
     .control {
-      display: none;
+      background-color: #3c3c3c;
+      bottom: 100%;
+      left: 0;
+      margin: 0;
+      position: absolute;
```

Comment:
Properties should be ordered background-color, bottom, left, margin, opacity, position, transition, width, z-index